scss/civicrm/common/_radio-checkbox.scss

```diff
@@ -1,3 +1,60 @@
+%checkbox-styled {
```

Comment:
use mixings, avoid using placeholders that force you to use `@extend`

Reply:
ok